src/main/java/com/fasterxml/jackson/databind/jsonFormatVisitors/JsonFormatTypes.java

```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
 
+//TODO what to do here
```

Comment:
I'm not sure how to approach this one. We can't really pass the locale here.
But at the same time it's not causing (known) problems yet